sys/riscv/conf/FETT

```diff
@@ -0,0 +1,15 @@
+#NO_UNIVERSE
+
+include "GENERIC"
+
+ident FETT
+
+device		virtio_random	# VirtIO Entropy device
```

Comment:
Do we want this on at the moment given the corruption you were seeing?

Reply:
I've been disabling it in the host software, not in the kernel. A modules build would still include this, too, this is just so it's in the kernel. And the last boot I did had a kernel panic even without the entropy device present at run time, so its presence isn't the problem (even if it may exacerbate it).